src/AssemblerOptions.cpp

```diff
@@ -201,6 +201,11 @@ void AssemblerOptions::addConfigurableOptions()
         default_value(10000),
         "Read length cutoff. Shorter reads are discarded.")
 
+        ("Reads.desiredCoverage",
+        value<uint64_t>(&readsOptions.desiredCoverage)->
+        default_value(0),
+        "Desired coverage as a number of bases. Coverage will be reduced to this if necessary.")
```

Comment:
After the PR is merged, I will update the documentation and `shasta.conf` to reflect this new option. I will also expand this help message a bit, consistently with the documentation changes, to explain the meaning of specifying zero coverage.

Reply:
Good point. The assumption is that this flag will be set by the `GenerateConfig.py` script, which asks the user for genome size in mega bases. I do not anticipate users entering this value manually.

Reply:
There will certainly be cases where users don't want to use `GenerateConfig.py` for whatever reason, but want to be able to specify desired coverage manually (in bases, without reference to a genome length). This would be a very useful option that I have missed many times when running Shasta assemblies. Requiring all the zeros would make it less usable but still useful.

Reply:
Makes sense. I will support suffixes in the other PR I put up about this change (Where we swap the entire Reads object in the Assembler). Should we support anything other than `G/Gbp` and `M/Mbp`?

Reply:
I would also do `K` - it could be useful for very small viral genomes. Although in that case that option is not as needed as for the longer genomes, because there would not be too many zeros. 

If you allow `Gbp/Mbp/Kbp`, I would also allow `Gb/Mb/Kb`. 

Those prefixes should be interpreted as powers of 10, not powers of 1024. I will clarify that in the documentation too.